client/extension.ts

```diff
@@ -24,6 +24,14 @@ import { LanguageClient, LanguageClientOptions, ServerOptions, TransportKind } f
 import { ClientResponseProvider } from './clientResponseProvider';
 import { ConfigurationManager } from './configurationManager';
 import { EditorQueryType, EditorQueryEvaluator } from './editorQueryEvaluator';
+import { ClientContext } from './marklogicClient';
```

Comment:
Is this a fix for something, or was it not meant to be included? Same goes for the xqyFormatting changes too. Was only expecting files in the ./docs directory.

Reply:
Those changes have been removed.